doc/classes/Control.xml

```diff
@@ -493,10 +493,14 @@
 			<return type="Vector2" />
 			<description>
 				Returns the position of this [Control] in global screen coordinates (i.e. taking window position into account). Mostly useful for editor plugins.
-				Equals to [member global_position] if the window is embedded (see [member Viewport.gui_embed_subwindows]).
+				Equivalent to [code]get_screen_transform().origin[/code] in all cases (see [method CanvasItem.get_screen_transform]), and equivalent to [code]get_global_transform_with_canvas().origin[/code] if the window is embedded (see [member Viewport.gui_embed_subwindows]).
```

Comment:
Actually, I think the second part:
```
equivalent to [code]get_global_transform_with_canvas().origin[/code] if the window is embedded (see [member Viewport.gui_embed_subwindows])
```
is not needed at all. It only adds unnecessary noise. `get_screen_position()` is a shorthand for `get_screen_transform().origin`, it's as simple as that.
The embedded window detail (and whatever other details) can/should be explained in the linked `[method CanvasItem.get_screen_transform]` docs (it is mentioned there indeed).

So I think this would be sufficient:
```suggestion
				Equivalent to [code]get_screen_transform().origin[/code] (see [method CanvasItem.get_screen_transform]).
```